File: PHP/core/week-1/c-programs/programs/22.php
<?php
    $num = 28;
    $sum = 0;

    for ($i = 1; $i <= $num / 2; $i++) {
        if ($num % $i == 0) {
            $sum += $i;
        }
    }

    if ($sum == $num) {
        echo "$num is a perfect number.";
    } else {
        echo "$num is not a perfect number.";
    }
?>

File: One Task/Task 1/c programming/22.c
// Perfect Number Check

#include <stdio.h>

int main() {
    int n = 6, sum = 0;
    if(n > 1) {
        for(int i = 1; i<n; i++) {
            if(n%i == 0) {
                sum += i;
            }
        }
        if(sum == n) {
            printf("%d is perfect number", n);
        }else {
            printf("%d is not a perfect number", n);
        }
    }else {
        printf("%d is not a perfect number", n);
    }
    return 0;
}